src/downshift.js

```diff
@@ -287,7 +291,10 @@ class Downshift extends Component {
             : this.props.itemToString(item),
         ...otherStateToSet,
       },
-      cbToCb(cb),
+      () => {
+        this.focusInput()
+        cbToCb(cb)()
```

Comment:
Let's forward the arguments and we don't need the cbToCb utility in this situation:

```js
(...args) => {
  this.focusInput()
  typeof cb === 'function' && cb(...args)
}
```